kalitectl.py

```diff
@@ -400,7 +400,7 @@ def start_watchify():
     sys.stdout.write('Starting watchify')
 
     watchify_process = subprocess.Popen(
-        ['node compile_javascript.js --debug --watch --staticfiles'],
+        args='node compile_javascript.js --debug --watch --staticfiles',
```

Comment:
But is it necessary for it to work on Windows? Or was that the part where you changed the list to a string?